src/index.js

```diff
@@ -9,6 +9,11 @@ import OneGraphApolloClient from "onegraph-apollo-client";
 import {ApolloProvider} from "react-apollo";
 import api from "./lib/apiGraphQL";
 import {getAppVersion} from "./lib/appVersion";
+import * as Sentry from "@sentry/react";
+import {isProd} from "./lib/isProd";
+
+isProd() &&
+  Sentry.init({dsn: "https://33235628c0f14f559dc62e408e3b4299@o180224.ingest.sentry.io/5311329"});
```

Comment:
```suggestion
  Sentry.init({dsn: "https://c38b324e1edf48e08ce298a649417905@o416962.ingest.sentry.io/5314544"});
```
Adding the new sentry "sponsored" account

Reply:
Awesome 🎉